Makefile

```diff
@@ -739,12 +739,17 @@ ggml-backend.o: ggml-backend.c ggml.h ggml-backend.h
 ggml-quants.o: ggml-quants.c ggml.h ggml-quants.h ggml-common.h
 	$(CC) $(CFLAGS)    -c $< -o $@
 
+ggml-aarch64.o: ggml-aarch64.c ggml.h ggml-aarch64.h ggml-common.h
+	$(CC) $(CFLAGS)    -c $< -o $@
+
 unicode.o: unicode.cpp unicode.h
 	$(CXX) $(CXXFLAGS) -c $< -o $@
 
 unicode-data.o: unicode-data.cpp unicode-data.h
 	$(CXX) $(CXXFLAGS) -c $< -o $@
 
+OBJS += ggml-alloc.o ggml-backend.o ggml-quants.o unicode.o unicode-data.o ggml-aarch64.o
```

Comment:
This line is not needed - instead update the one above

Reply:
We removed this line in the latest commit and included `ggml-aarch64.o` in the previous OBJS, as suggested.